coderd/workspaceagents_test.go

```diff
@@ -1623,7 +1623,8 @@ func TestWorkspaceAgentExternalAuthListen(t *testing.T) {
 			ticks <- time.Now()
 		}
 		cancel()
-		// We expect only 1
+		// We expect only 2. One from the initial "Refresh" attempt, and
+		// another from the first tick. Ideally this would be just one.
 		// In a failed test, you will likely see 9, as the last one
 		// gets canceled.
 		require.Equal(t, 1, validateCalls, "validate calls duplicated on same token")
```

Comment:
Comment does not agree with below `require.Equal` -- do we only expect 1 or 2?

Reply:
Fixed the typo...